x-pack/plugins/observability_solution/logs_shared/public/hooks/use_kibana.tsx

```diff
@@ -20,8 +20,7 @@ import {
 } from '../types';
 
 export type PluginKibanaContextValue = CoreStart &
-  LogsSharedClientStartDeps &
-  LogsSharedClientStartExports;
+  LogsSharedClientStartDeps & { logsShared: LogsSharedClientStartExports };
```

Comment:
This aligns the shape of the context with that of the other consuming plugins. This way no distinction of local and non-local usage is necessary at the usage site.